go.sum

```diff
@@ -309,7 +309,6 @@ github.com/go-errors/errors v1.0.1/go.mod h1:f4zRHt4oKfwPJE5k8C9vpYG+aDHdBFUsgrm
 github.com/go-gl/glfw v0.0.0-20190409004039-e6da0acd62b1/go.mod h1:vR7hzQXu2zJy9AVAgeJqvqgH9Q5CA+iKCZ2gyEVpxRU=
 github.com/go-gl/glfw/v3.3/glfw v0.0.0-20191125211704-12ad95a8df72/go.mod h1:tQ2UAYgL5IevRw8kRxooKSPJfGvJ9fJQFa0TUsXzTg8=
 github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200222043503-6f7a984d4dc4/go.mod h1:tQ2UAYgL5IevRw8kRxooKSPJfGvJ9fJQFa0TUsXzTg8=
-github.com/go-gorp/gorp/v3 v3.0.2 h1:ULqJXIekoqMx29FI5ekXXFoH1dT2Vc8UhnRzBg+Emz4=
```

Comment:
question: I'm confused as to why there are modifications to `go.sum` here without any corresponding changes to `go.mod`.

I can't reproduce this with `go mod tidy` locally, either. Am I missing something here?

Reply:
And I don't have a proxy set or anything such. I guess it might be a weirdness of my local environment given that neither you nor the last person who made changes to cert-manager got this diff. So perhaps I'll remove it now and investigate what's happening locally

Reply:
> So perhaps I'll remove it now

Done